.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Please, add the `test.md` file inside the .gitignore file after creating it as mentioned in the requirement of this project, the `.gitignore` file is super important in cases we don't want to push some files such as in our case `test.md` file by mistake ('Don't worry, everybody makes mistakes)